src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,70 @@
+import {useState} from 'react';
+import Parse from 'parse';
+import { useSelector } from 'react-redux';
+import { ApplicationState } from 'types/store';
+import MenuButton from 'components/Menu/MenuItem/MenuButton';
+import MenuToggle from 'components/Menu/MenuItem/MenuToggle';
+import { ReactComponent as AddImageIcon } from 'assets/icon-addimage.svg';
+import { ReactComponent as AddStickerIcon } from 'assets/icon-addsticker.svg';
+import { ReactComponent as CheckIcon } from 'assets/icon-check.svg';
+import { ReactComponent as SettingsIcon } from 'assets/icon-settings.svg';
+import { ReactComponent as ColumnIcon } from 'assets/icon-column.svg';
+import { ReactComponent as FocusIcon } from 'assets/icon-focus.svg';
+import { ReactComponent as TimerIcon } from 'assets/icon-timer.svg';
+import { ReactComponent as VoteIcon } from 'assets/icon-vote.svg';
+import { ReactComponent as ToggleSettingsMenuIcon } from 'assets/icon-toggle-settings-menu.svg';
+import { ReactComponent as ToggleAddMenuIcon } from 'assets/icon-toggle-add-menu.svg';
+import './Menu.scss';
+
+function Menu() {
+
+    const [showAdminMenu, toggleMenus] = useState(false); 
+
+    const currentUser = Parse.User.current();
+    const admins = useSelector((state: ApplicationState) => state.users.admins);
+
+    const isAdmin = admins.map(admin => admin.id).indexOf(currentUser!.id) !== -1;
```

Comment:
I'm not sure if the result is always `-1` in each browser or if it's just a negative number by the specification. Can you look it up or are you sure that it's `-1` specified?

Reply:
According to https://developer.mozilla.org it's always -1 in case the elements isn't present